src/Microsoft.VisualStudio.Validation/Assumes.InternalErrorException.cs

```diff
@@ -1,6 +1,6 @@
 /********************************************************
 *                                                        *
-*   � Copyright (C) Microsoft. All rights reserved.      *
+*   � Copyright (C) Microsoft. All rights reserved.      *
```

Comment:
? did you mean to make this change?

Reply:
Good catch. Sure didn't mean to make _that_ change. Fixed.